c_src/build_deps.sh

```diff
@@ -8,7 +8,7 @@ if [ `uname -s` = 'SunOS' -a "${POSIX_SHELL}" != "true" ]; then
 fi
 unset POSIX_SHELL # clear it so if we invoke other scripts, they run as ksh as well
 
-LEVELDB_VSN="end-to-end/timeseries"
+LEVELDB_VSN="tags/2.1.3"
```

Comment:
Let's change this to "2.0.9".  That version has all the sequential data improvements from "develop" branch.  2.0 and 2.1 branches have same API at this time.